cbz_tagger/entities/volume_entity.py

```diff
@@ -86,6 +86,9 @@ def get_volume(self, chapter_number: str) -> str:
         if len(self.volume_map) == 0:
             return "-1"
 
+        if len(self.volume_map) == 1 and self.volume_map[0][0] == "-1":
```

Comment:
This situation can occur if. there is a valid volume respoonse, but no usable numerical volume entities are present